src/sentry/static/sentry/app/components/events/interfaces/utils.jsx

```diff
@@ -38,7 +39,7 @@ export function getCurlCommand(data) {
         result += ' \\\n --data "' + escapeQuotes(JSON.stringify(data.data)) + '"';
         break;
       case 'application/x-www-form-urlencoded':
-        result += ' \\\n --data "' + escapeQuotes(jQuery.param(data.data)) + '"';
+        result += ' \\\n --data "' + escapeQuotes(queryString.stringify(data.data)) + '"';
```

Comment:
I think this will change the output when a parameter is used multiple times. `jQuery.param()` would create `foo[]=` while `query-string` will create `foo=`. In some frameworks/languages this is a meaningful difference.